all/Dockerfile

```diff
@@ -21,6 +21,7 @@ COPY install-protobuf.sh /tmp
 RUN chmod +x /tmp/install-protobuf.sh
 RUN /tmp/install-protobuf.sh
 RUN git clone https://github.com/googleapis/googleapis
+RUN git clone https://github.com/grpc-ecosystem/grpc-gateway
```

Comment:
this should already be present in the final container because we `go get` it. You can copy the files into the include directory from `/go/src/github.com/grpc-ecosystem/grpc-gateway` to the include dir with a vanilla `RUN cp` I believe